docs/documenting-schema.rst

```diff
@@ -0,0 +1,84 @@
+.. _documenting-schema:
+
+Documenting schema
+==================
+
+You can improve the experience of consuming your GraphQL API by documenting the types that it supports.
```

Comment:
I've been thinking about this section for a while now, and I've reached the conclusion that it should pivot it's attention to the fact in GraphQL API exploration and documentation is part of the standard.

Something along the lines of "One of GraphQL features that improves developer experience is build-in support for schema exploration and documentation generation."  We can then give more info on each of details in next sections.

Reply:
I took a stab at re-organizing the article to focus on the GraphQL standard and its motives.

I feels to me like it gives a little less opportunity to say: "GraphQL descriptions are a thing.  Ariadne supports them.  Use GraphQL SDL to define them in Ariadne."  I think it might read a little less like documentation for an Ariadne feature and more like a sales pitch for GraphQL?  😅

I kind of liked your previous outline better, but what do you think?